doc/getting-started.rst

```diff
@@ -332,7 +332,7 @@ specific parser. Here's a small C++ program that parses input with our
 
 .. code::
 
-    # spicy-build -o a.out my-http.cc my-http.spicy
+    # spicy-build -o a.out -S my-http.cc my-http.spicy
```

Comment:
Did you find you need the `-S` to make this work? It's should actually be as shown: no option to automatically include the spicy-driver C++ skeleton; and with `-S` to *not* include it, which would mean that you can't actually have the resulting `a.out` parse anything. (It used to be opposite in an older version, but that ended up just being confusing because it made the most common use case require an option).

Reply:
yeah.. I believe I was getting duplicate 'main' functions during build if I didn't use -S.. i'll have to test it again though.

Reply:
I'm wondering if that's another linker issue, there're a weak main() symbols involved ...

Reply:
```
justin@ap1k:~/src/spicy/doc/examples$ spicy-build -o a.out my-http.cc my-http.spicy
/nix/store/nyhj00w339gk2gaj3faz70gjrnbmam8v-binutils-2.31.1/bin/ld: /run/user/1000/ccVnoX2q.o: in function `main':
my-http.cc:(.text.startup+0x0): multiple definition of `main'; /run/user/1000/ccMa3gxn.o:spicy-driver-host.cc:(.text.startup+0x0): first defined here
collect2: error: ld returned 1 exit status
```

but with -S it works

```
justin@ap1k:~/src/spicy/doc/examples$ spicy-build -o a.out -S my-http.cc my-http.spicy
justin@ap1k:~/src/spicy/doc/examples$ echo "GET /index.html HTTP/1.0"  | ./a.out
GET, /index.html, 1.0
```

Reply:
I can't reproduce this. it's pretty certainly an artifact of the linking, and the docs describe how it's supposed to work, so I'll close this.